.github/workflows/update_client.yaml

```diff
@@ -0,0 +1,43 @@
+name: Update SDK
+
+# Define the events that trigger the workflow
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "api/**"
+
+jobs:
+  update_api:
+    # Define the OS and environment the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+      # Step 1: Checkout the first repository
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Step 2: Checkout the second repository
+      - name: Checkout second repository
+        uses: actions/checkout@v4
+        with:
+          repository: wordlift/python-client
+          path: "./python-client"
+
+      # step 3: Patch middleware api specification
+      - name: Apply patch to middleware.yaml
```

Comment:
I am undecided whether this patch should go when the docs repo receives yaml updates independently or go to the python client repo.

I am leaning towards the first, that is the docs repo should patch the yaml independently.

What do you think?

Reply:
both are acceptable, but i prefer that docs sends a correct version of the apis and in general to use the docs as reference.